torchaudio/functional/functional.py

```diff
@@ -1444,3 +1446,45 @@ def resample(
                                               resampling_method, beta, waveform.device, waveform.dtype)
     resampled = _apply_sinc_resample_kernel(waveform, orig_freq, new_freq, gcd, kernel, width)
     return resampled
+
```

Comment:
There should be two lines between function definitions.

Reply:
I've addressed this [here](https://github.com/pytorch/audio/pull/1601/files#diff-cf61df2ded7de3af992d120c6b1530bddb6333ad62e825ce4c9251dafe9afd1dR422). I've also run `flake8` and added one more space before inline comments [here](https://github.com/pytorch/audio/pull/1601/files#diff-cf61df2ded7de3af992d120c6b1530bddb6333ad62e825ce4c9251dafe9afd1dR407) and change the list comprehension to list function [here](https://github.com/pytorch/audio/pull/1601/files#diff-cf61df2ded7de3af992d120c6b1530bddb6333ad62e825ce4c9251dafe9afd1dR407). Please check.